to-do/main.js

```diff
@@ -0,0 +1,68 @@
+const STATUS = {
+	TO_DO: 'To Do',
+	IN_PROGRESS: 'In Progress',
+	DONE: 'Done',
+}
+const PRIORITY = {
+	HIGH: 'high',
+	LOW: 'low',
+}
+
+let toDoList = [
+	{ task: 'learn objects', status: STATUS.DONE, priority: PRIORITY.HIGH },
+	{ task: 'learn arrays', status: STATUS.IN_PROGRESS, priority: PRIORITY.HIGH },
+	{ task: 'take a rest', status: STATUS.TO_DO, priority: PRIORITY.LOW },
+]
+
+function addTask(
+	taskParameter,
+	statusParameter = STATUS.TO_DO,
+	priorityParameter = PRIORITY.LOW
+) {
+	let independentCase = taskParameter.toLowerCase()
+	let findMethod = toDoList.find(element => element.task === independentCase)
```

Comment:
Как вариант к 1-му пункту:
```suggestion
	let unifiedString = taskParameter.toLowerCase()
	let isTaskExist = toDoList.find(element => element.task === independentCase)
```